COMRS-194 Generate CSV exports for FICHE DU PATIENT AVEC VIH - ADULTE in Initializer module formats

lauravignoli opened the pull request:
https://jembiprojects.jira.com/browse/COMRS-194

lauravignoli committed:
COMRS-194 Add adult hiv form concepts

lauravignoli committed:
COMRS-194 Add adult hiv treatment form concepts

lauravignoli committed:
COMRS-194 Add child hiv form concepts

tmvumbi2 commented on configuration/concepts/adult_hiv_initial_form.csv:
```diff
@@ -0,0 +1,97 @@
+7acfafa4-f19b-485e-97a7-c9e002dbe37a,,,Positif,Positive,,,Misc,N/A,,
+718b4589-2a11-4355-b8dc-aa668a93e098,,,Négatif,Negative,,,Misc,N/A,,
+b09900a6-4646-4804-b460-63be72879a13,,,Ne sait pas,Do not know,,,Misc,N/A,,
+351d2ae2-597e-471d-859c-ac0f782e06ec,,Positive;Negative;Do Not Know,Statut Partenaire,Status Partner,,,Misc,Coded,,
```
@lauravignoli Could you please double check the above concept? Noticed the following issues:
`Fully specified name:fr` = `Positive;Negative` (the text is in English)
The corresponding English translation (`Do Not Know`) does not look right.
And also the short name seems to be unrelated to the fully specified name.

lauravignoli replied:
@tmvumbi2 This is working as expected to me? The possible answers are Positive, Negative or Do Not Know.

![statusPartner](https://user-images.githubusercontent.com/1198827/54992518-2c7db480-4fc8-11e9-8356-a3b0da0f6839.png)

tmvumbi2 replied:
@lauravignoli oh you right, I didn't see that this was an answer list

tmvumbi2 commented on configuration/concepts/adult_hiv_initial_form.csv:
```diff
@@ -0,0 +1,97 @@
+3b75cc72-924b-4d36-8891-95484287495e,,,Inconu,Unknown,,,Misc,N/A,,
+9d26a59c-fe56-4509-b931-e380303a72c3,,,Non renseigné,Not specified,,,Misc,N/A,,
+2d1bf419-16a9-44a7-a11c-98d39b170cec,,a2065636-5326-40f5-aed6-0cc2cca81ccc;b497171e-0410-4d8d-bbd4-7e1a8f8b504e;Unknown;9d26a59c-fe56-4509-b931-e380303a72c3,Couples discordants,Discordant couples ,,,Misc,Coded,,
+d2b400ea-7510-432c-9ea4-27ea88736bb7,,a2065636-5326-40f5-aed6-0cc2cca81ccc;b497171e-0410-4d8d-bbd4-7e1a8f8b504e; ,Utilisation systématique du préservatif,Condom use systematically,,,Misc,Coded,,
```
@lauravignoli Could you please remove the trailing ";" at the end of `b497171e-0410-4d8d-bbd4-7e1a8f8b504e; ,`

tmvumbi2 commented on configuration/concepts/adult_hiv_initial_form.csv:
```diff
@@ -0,0 +1,97 @@
+76cab961-e2ad-4d2f-bb00-03c73ca7ce11,,,VIH1P,HIV1P,,,Misc,N/A,,
+965b14e1-fbdb-4590-9e24-9fa5a64c68ce,,597f78bc-09ac-4473-b686-7d81b26cf64d;92cea782-a50b-439c-a3ae-1acc785c2c42;de0efbb3-4f8c-4054-b834-23c2f8b75112;ca6e7017-241f-4d5e-ba17-efd280c1b6cd;4cd723ee-83d1-4db8-86ca-eab4e7d220b0;76cab961-e2ad-4d2f-bb00-03c73ca7ce11,Type VIH,Type HIV,,,Misc,Coded,,
+23d94813-c123-4f6d-a7f8-66e22bc30e75,0d9692be-a4ea-48b8-bf44-3c6c8eb7ecf5;dcc5a64e-bf9f-402e-8ce3-42ccd3fb83ce;faa8bd18-23b0-4988-9452-1a60094f50fa;965b14e1-fbdb-4590-9e24-9fa5a64c68ce,,III.1 Test VIH,III.1 HIV TEST,,,Misc,N/A,,
+c6faef09-0a6c-4f3b-98c6-a47c7299cf5b,,a2065636-5326-40f5-aed6-0cc2cca81ccc;b497171e-0410-4d8d-bbd4-7e1a8f8b504e; ,Maladies opportunistes,Opportunitis disease,,,Misc,Coded,,
```
@lauravignoli small typo  `Opportunitis` -> `Opportunistic`

tmvumbi2 commented on configuration/concepts/adult_hiv_initial_form.csv:
```diff
@@ -0,0 +1,97 @@
+e765f0df-ab4d-47a8-a51d-6ca9277cae32,,,D,D,,,Misc,N/A,,
+56516172-9d35-4e52-9d98-00ff82235aa6,,f6ddc619-4093-4077-b91a-ad54306243e1;ceb11aff-09b7-4b1c-8ea8-7f98ef6079b1;e765f0df-ab4d-47a8-a51d-6ca9277cae32,Hépatites,Hepatitis(s),,,Misc,Coded,,
+83624a21-972b-40e2-af3e-1a42612f4756,,a2065636-5326-40f5-aed6-0cc2cca81ccc;b497171e-0410-4d8d-bbd4-7e1a8f8b504e,Autres maladies,Other diseases,,,Misc,Coded,,
+9200f505-f983-442b-99e7-23430e26e489,,,Drogue IV,Drogue IV,,,Misc,N/A,,
```
@lauravignoli Was the English translation meant to be `Drug  IV`?

tmvumbi2 commented on configuration/concepts/adult_hiv_initial_form.csv:
```diff
@@ -0,0 +1,97 @@
+56516172-9d35-4e52-9d98-00ff82235aa6,,f6ddc619-4093-4077-b91a-ad54306243e1;ceb11aff-09b7-4b1c-8ea8-7f98ef6079b1;e765f0df-ab4d-47a8-a51d-6ca9277cae32,Hépatites,Hepatitis(s),,,Misc,Coded,,
+83624a21-972b-40e2-af3e-1a42612f4756,,a2065636-5326-40f5-aed6-0cc2cca81ccc;b497171e-0410-4d8d-bbd4-7e1a8f8b504e,Autres maladies,Other diseases,,,Misc,Coded,,
+9200f505-f983-442b-99e7-23430e26e489,,,Drogue IV,Drogue IV,,,Misc,N/A,,
+a32c47d6-ff4b-4f0b-b8f6-dea611f17f59,,9200f505-f983-442b-99e7-23430e26e489,Toxicology,Toxicology,,,Misc,Coded,,
```
@lauravignoli The French translation should read `Toxicologie`

tmvumbi2 commented on configuration/concepts/adult_hiv_initial_form.csv:
```diff
@@ -0,0 +1,97 @@
+fef74daa-2a47-4c4e-b9c2-d18078d8e3be,,,Date de fin (le cas échéant),End Date  ,,,Misc,Date,,
+81ff1578-da33-44f3-bdbe-71d7706ecd87,,,Formation sanitaire,Health Facility,,,Misc,Text,,
+09704fda-7fe3-45bf-a993-9b5eb7b69ed9,,,Transfert avec dossier,Transfer with file,,,Misc,N/A,,
+c74e2fd5-1005-49e8-bf2a-64aabd2c662a,,,Déjé été sou ARV,Already been ARV,,,Misc,N/A,,
```
@lauravignoli Small typo on the french translation: `Déjé été sou ARV` -> `Déjà été sous ARV`

tmvumbi2 commented on configuration/concepts/adult_hiv_initial_form.csv:
```diff
@@ -0,0 +1,97 @@
+aeecf2e8-4746-4023-bc39-efdd83ce4e94,,09704fda-7fe3-45bf-a993-9b5eb7b69ed9;c74e2fd5-1005-49e8-bf2a-64aabd2c662a;51d7342d-d8ad-4061-97f3-c32724a47694,Circonstances du traitement ARV antérieur,Circumstances of previous ARV treatment,,,Misc,Coded,,
+3b2a2ded-60f6-4818-a4f4-10f296f53ae1,,,Médicament ARV prescrits,Prescribed ARV drug,,,Misc,Text,,
+29538d59-7239-475d-b95c-3e493f8b88fe,,a2065636-5326-40f5-aed6-0cc2cca81ccc;b497171e-0410-4d8d-bbd4-7e1a8f8b504e,Traitement toujours en cours,Treatment still in progress,,,Misc,Coded,,
+db3088d9-cc5c-433f-8e47-5e29881dd9c5,,a2065636-5326-40f5-aed6-0cc2cca81ccc;b497171e-0410-4d8d-bbd4-7e1a8f8b504e;Do Not Know,Cotrimoxazole (Prophylaxie en cours),Cotrimoxazole (Prophylaxsis in progress),,,Misc,Coded,,
```
@lauravignoli  Could you please double check the above concept? Noticed the following issues:
`Fully specified name:fr` =`Do Not Know` : the text is in English
`Fully specified name:en` = `Cotrimoxazole (Prophylaxie en cours)`: the text is in French and seems unrelated to its fr counterpart
`Short name:fr` =`Cotrimoxazole (Prophylaxsis in progress)`: the text is english
`Short name:en` is empty

lauravignoli replied:
I decided to use some concepts with the name instead of uuid after I had to update the uuid because it was a duplicate. When there's no duplicates (which should be always) uuid or concept name can be used in-distinctively.

Remember that in this file, for the name, the first column is fr and the second en. I removed the short names when they are the same as the full name.

tmvumbi2 replied:
Thanks for the clarification @lauravignoli, I missed that  'Do Not Know' was part of the answer

tmvumbi2 commented on configuration/concepts/adult_hiv_initial_form.csv:
```diff
@@ -0,0 +1,97 @@
+20b2eac4-ff3a-4569-b610-067c4cd8b027,e3f9c7ee-aa3e-4224-9d18-42e09b095ac6,,Traitements,Treatments,,,Misc,N/A,,
+059b8309-4cba-46cc-a857-e1e74be3fa86,20b2eac4-ff3a-4569-b610-067c4cd8b027,,IV. VISITE INITIALE,IV. INITIAL VISIT,,,Misc,N/A,,
+1fb2dd86-53b5-4815-9c64-edc081b908d9,83e4df0e-5723-4c53-ae8b-7411b2337bef;7d1187b3-f428-4064-80ec-869c16761793;059b8309-4cba-46cc-a857-e1e74be3fa86,,FICHE DU PATIENT AVEC VIH - ADULTE,PATIENT WITH HIV - ADULT INITIAL,,,Misc,N/A,,
+9d91cb77-3f10-11e4-adec-0800271c1b75,18351dd1-078e-456c-8c23-915b16e9dfaf;c36a7537-3f10-11e4-adec-0800271c1b75;1fb2dd86-53b5-4815-9c64-edc081b908d9,,All Observation Templates,All Observation Templates[FR],,,ConvSet,N/A,,
```
@lauravignoli this change will be overridden in `/child_hiv_initial_form.csv`, I would like to suggest we remove it here

lauravignoli committed:
COMRS-194 Fixed typos and removed unnecessary line

Removed the last line in adult hiv file as it will be overridden by child hiv file.
